mpl_infer_tiles_GPU_new.py

```diff
@@ -21,17 +23,28 @@
 from mpl_config import MPL_Config, PolygonConfig
 from skimage.measure import find_contours
 
-class Predictor(multiprocessing.Process):
-    def __init__(self, input_queue, gpu_id,
-                          POLYGON_DIR,
-                          weights_path,
-                          output_shp_root,
-                          x_resolution,
-                          y_resolution,len_imgs,image_name):
 
+class Predictor(multiprocessing.Process):
+    def __init__(
+        self,
+        config: MPL_Config,
+        input_queue: multiprocessing.JoinableQueue,
+        use_gpu: bool,
```

Comment:
similarly - could we look at the number of gpus variable in the config?

Reply:
Done. Lmk what you think!